atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraVerifier.java

```diff
@@ -342,17 +342,4 @@ private static void checkRfsMatchConfig(KsDef ks, CassandraKeyValueServiceConfig
             }
         }
     }
-
-    static final FunctionCheckedException<CassandraClient, Boolean, UnsupportedOperationException>
```

Comment:
can we actually delete this? aren't there other tables we CAS?

Reply:
We do use CAS elsewhere, but we don't actually check that it is supported and CassandraKvsImpl.supportsCheckAndSet() returns true regardless of what happens during intitialisation. We should then either wire it up correctly or just remove it since no one is afaik on a version of Cassandra that doesn't support it.

Reply:
yea it was mostly a concern back when we had PG impls using cassandra 1.2, but my guess is it's fine to remove at this point